program/convert-minutes-to-hours/convert_minutes_to_hours.go

```diff
@@ -0,0 +1,14 @@
+package main
+import ("fmt"
+        "math")
+
+func minutesToHours(nums int) float64 {
+  hours := float64(nums) / 60;
+  // round to get 2 digits
+  return math.Round(hours * 100) / 100;
+}
+
+func main() {
+  test := minutesToHours(80)
```

Comment:
This program is returning 80 minutes as 1.33 hour please do check the logic
<img width="217" alt="Screenshot 2022-10-15 at 12 48 08 AM" src="https://user-images.githubusercontent.com/79962713/195924851-bc236270-97e0-4185-9c14-a2355a9a9c98.png">

Reply:
80/60 = 1.333333333